src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/QueueReleaseFunction.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Model;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Services.Interfaces;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using static GovUk.Education.ExploreEducationStatistics.Publisher.Model.ReleaseInfoStatus;
+
+namespace GovUk.Education.ExploreEducationStatistics.Publisher.Functions
+{
+    public class QueueReleaseFunction
```

Comment:
Would be good to get a comment at the top here to explain the function of this class

Reply:
I have added it just above the function